pycode/memilio-epidata/README.rst

```diff
@@ -93,119 +53,25 @@ Required python packages:
-
-optional arguments working for some are:
-
-+---------------------------------------------+-----------------------------------------------------------+
-| -p, --make-plot                             | Plots the data.                                           |
-+---------------------------------------------+-----------------------------------------------------------+
-| -ed, --end-date                             | Changes date for which data collection is stopped [divi]  |
-+---------------------------------------------+-----------------------------------------------------------+
-| -sd, --start-date                           | Changes date for which data collection is started [divi]  |
-+---------------------------------------------+-----------------------------------------------------------+
-| -fd, --fill-dates                           | Returns dataframes with all dates instead of only dates   |
-|                                             | where new cases have been reported.                       |
-|                                             |  Note that this option will have a negative impact        |
-|                                             |  on performance as well as on the storage space needed.   |
-|                                             |  [cases]                                                    |
-+---------------------------------------------+-----------------------------------------------------------+
-| -m N, --moving-average N                       | The central N days moving average is computed for the data.        |
-|                                             |  Note that the --impute_dates option will be implicitly     |
-|                                             |  turned on, as computing the moving average requires all  |
-|                                             |  dates to be available. [cases]                             |
-+---------------------------------------------+-----------------------------------------------------------+
-| -sb, --split-berlin                         | Berlin data is split into different counties              |
-|                                             |  , instead of having only one county for Berlin. [cases]    |
-+---------------------------------------------+-----------------------------------------------------------+
-| -u, -- update-data                          | Just chronological missing data is added,                 |
-|                                             | **after** the existing ones [divi]                        |
-+---------------------------------------------+-----------------------------------------------------------+
-
-Hint:
-When using the "--make-plot" option close one figure-window to get the next one.
-
-Results
--------
+To run the scripts use the setup.py and everything is installed (see Installation) and useable via the following entry points.
```

Comment:
I think this sentence could be smoother and shorter. Installation was already presented and setup.py already mentioned. Also there are two and's. Maybe something like:
'After installation run the scripts via the following entry points.'